lms/urls.py

```diff
@@ -922,11 +922,11 @@
 
 if settings.DEBUG:
     urlpatterns += static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
-    urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
     urlpatterns += static(
```

Comment:
Maybe add something like the following to keep this from regressing?
```suggestion
    # profile image urls must come before the media url to work
    urlpatterns += static(
```